orchard-provider-aws/src/main/resources/reference.conf

```diff
@@ -1,5 +1,15 @@
 com.salesforce.mce.orchard {
     io.aws {
         logging.uri = ${?ORCHARD_AWS_LOGGING_URI}
+        static.credentials {
+            enabled = false
```

Comment:
don't think we need this, can use the presence of the configuration as `enabled`

Reply:
@realstraw let me change